tests/trace/test_weave_client.py

```diff
@@ -2389,3 +2390,135 @@ def test_calls_query_sort_by_display_name_prioritized(client):
 
     # Verify they all have the same op_name
     assert call_list[0].op_name == call_list[1].op_name == call_list[2].op_name
+
+
+def test_calls_descendants_basic(client):
+    """Test basic functionality of calls_descendants including depth limits and multiple children."""
+
+    @weave.op()
+    def parent_op(x: int) -> int:
+        child1 = child_op1(x)
+        child2 = child_op2(x)
+        return child1 + child2
+
+    @weave.op()
+    def child_op1(x: int) -> int:
+        return grandchild_op(x)
+
+    @weave.op()
+    def child_op2(x: int) -> int:
+        return x * 3
+
+    @weave.op()
+    def grandchild_op(x: int) -> int:
+        return x * 2
+
+    # Create a call that will have multiple children and grandchildren
+    result = parent_op(5)
+    parent_call = client.get_calls(filter=tsi.CallsFilter(op_names=["%parent_op:*"]))[0]
```

Comment:
What does the `%` mean here?

Reply:
this might actually be sqlinjection, lemme test something